Homework/Task2_3/Training.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task2_3
+{
+    public class Training : ICloneable
+    {
```

Comment:
array has to be used here -> TrainingContent[] contents

Reply:
Changed the clas to use an array instead of list